modules/otel-collector-components.adoc

```diff
@@ -557,7 +557,36 @@ config: |
 <2> The default exporter when the attribute value is not present in the table in the next section.
 <3> The table that defines which values are to be routed to which exporters.
 
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta processor
+
+The Cumulative to Delta processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+
+The Cumulative to Delta processor converts monotonic, cumulative sum and histogram metrics to monotonic, delta metrics.
+Non-monotonic sums and exponential histograms are excluded.
```

Comment:
@andreasgerstmayr, does this sentence mean that

A) The processor does not convert non-monolithic sums and exponential histograms
OR
B) The processor excludes non-monolithic sums and exponential histograms when it performs the conversion
?

Reply:
A